tests/e2e/api/routing.js

```diff
@@ -2,6 +2,7 @@ const _ = require('lodash');
 const utils = require('../../utils');
 const constants = require('../../constants');
 const moment = require('moment');
+const { expect } = require('chai');
```

Comment:
I see at some places we are using `chai.expect(...)` where as in some places (like here) we are importing `expect` specifically and then using it. 
1. Do we have data points as to why one would be better than the other
2. Whichever we choose it might be better to use that throughout to be consistent across the code base